src/fhir/PlanDefinition_cove.json

```diff
@@ -0,0 +1,438 @@
+      "id": "Day67Msg0",
+      "name": "Day 67 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 67,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "12:49:35"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "I try to look for something positive each day. Even though that some days I have to look a little harder. Hope you can find something positive in each day as well 😊"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day101Msg0",
+      "name": "Day 101 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 101,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "15:06:27"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Just wanted to remind you that you can start over, each morning. Setbacks happen! There will always be a new day on the horizon. 🌅"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day125Msg0",
+      "name": "Day 125 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 125,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "09:16:01"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Hey {name}, sending good thoughts and energy your way! ☀️✨ I care about how you’re doing. If you ever find yourself struggling, feel free to text me -Riley"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day162Msg0",
+      "name": "Day 162 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 162,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "13:39:35"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "I have a quote for you: \"The greatest glory in living lies not in never falling, but in rising every time we fall.\" (Nelson Mandela) Hope you are finding ways to keep getting back up! 🌱"
```

Comment:
@achen2401  will the parsers handle this escaped double quote okay? If not and it's complicated, then please switch these to single quotes.

Reply:
Looks fine to me in testing.